packages/datadog-plugin-oracledb/src/index.js

```diff
@@ -14,7 +15,13 @@ class OracledbPlugin extends Plugin {
 
     this.addSub('apm:oracledb:execute:start', ({ query, connAttrs }) => {
       const service = getServiceName(this.tracer, this.config, connAttrs)
-      const connectStringObj = new URL('http://' + connAttrs.connectString)
+      let connectStringObj
+      try {
+        connectStringObj = new URL(`http://${connAttrs.connectString}`)
```

Comment:
Would it be easier to extract only the host and the port? These are needed for the `out.*` tags (which also seem to be missing in the first place).

Reply:
You mean parsing the string? It's too complicated as it supports multiple address lists and description lists so there could be any number of connections listed in the connect descriptor and no way to know which one was actually used.

Reply:
Can we figure out somewhere else in the library which host is actually used?

Reply:
As far as I can tell, that happens in C somewhere. There's no way for us to get at it from JavaScript.

Reply:
Ok, then let's only add the tags for the happy path of a single host configured with a valid URL and otherwise not add it at all.

Reply:
Done.